### PR TITLE
Getting Started - Phase 2 

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This PR contains the current state of Getting Started Phase 2.

## Changed Pages

💻 [Getting Started](https://deploy-preview-3159--docs-spectrocloud.netlify.app/getting-started/)

## Jira Tickets

🎫 [DOC-855](https://spectrocloud.atlassian.net/browse/DOC-855)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-855]: https://spectrocloud.atlassian.net/browse/DOC-855?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/getting-started/additional-capabilities/additional-capabilities.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+> "Wouldn’t bringing some astronomy into everyone's home be great?" says Anya, Lead Astrophycist, who has always the
+> dreamer of the team. "I wonder how we could make that possible."
+>
+> Kai is in charge of scaling the Spacetastic platform. "That would be a great dream and a challenge for us, Anya." they
+> say.
+>
+> "You know, we might be able to make your dream happen!" says Wren, Founding Engineer. "Palette's edge capabilities
+> could make it possible for us to bring Spacetastic to many devices."
+>
+> "I can't believe my ears!" says Kai laughing. "Wren, our resident Palette sceptic, has well and truly embraced our new
```
Comment (lennessyy) [inline docs/docs-content/getting-started/additional-capabilities/additional-capabilities.md:43]:
```suggestion
> "I can't believe my ears!" says Kai laughing. "Wren, our resident Palette skeptic, has well and truly embraced our new
```

File: docs/docs-content/getting-started/aws/create-cluster-profile.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+to a host cluster onto AWS.
+
+## 🧑‍🚀 Catch up with Spacetastic
+
+Wren and Kai have created their first Palette cluster profile by following the steps described in this guide. They are
+in good spirits, as the process has gone smoothly.
+
+> "The visual representation of cluster profiles in Palette is much clearer than our whiteboard." says Kai, glancing
+> back at the list they have created. "I can keep track of which versions we are using in production just by reviewing
+> the profile. What are your thoughts, Wren? Have you remained a Palette sceptic?"
```
Comment (lennessyy) [inline docs/docs-content/getting-started/aws/create-cluster-profile.md:137]:
```suggestion
> the profile. What are your thoughts, Wren? Have you remained a Palette sceptic?"
```
```suggestion
> the profile. What are your thoughts, Wren? Have you remained a Palette skeptic?"
```

File: docs/docs-content/getting-started/aws/deploy-k8s-cluster.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+## 🧑‍🚀 Catch up with Spacetastic
+
+Wren and Kai have deployed their first cluster profile by following the steps described in this tutorial. They were
+impressed by how streamlined the process was and how the cluster profiles provided them with a deployment blueprint.
+
+> "Deploying our first cluster with Palette was intuitive." says Wren. "It's ideal to find an external partner that can
+> take care of our Kubernetes infrastructure and free us up to deliver more educational features. I definitely think
+> that Palette has the capabilities to take care of all the Kubernetes heavy lifting for us."
+>
+> "I agree with you and I'm glad to hear you're not as sceptical anymore." says Kai, nodding and laughing. "From a
```
Comment (lennessyy) [inline docs/docs-content/getting-started/aws/deploy-k8s-cluster.md:203]:
```suggestion
> "I agree with you and I'm glad to hear you're not as sceptical anymore." says Kai, nodding and laughing. "From a
```
```suggestion
> "I agree with you and I'm glad to hear you're not as skeptical anymore." says Kai, nodding and laughing. "From a
```

**Review by lennessyy — COMMENTED** [review]
Love the scenarios. At first i was a big Anya fan but it doesn't sound like she does much and i agree with Kai that she should give her team more vacation 

We are still using US English right? 😜

**Comment by addetz** [comment]
> Love the scenarios. At first i was a big Anya fan but it doesn't sound like she does much and i agree with Kai that she should give her team more vacation
> 
> We are still using US English right? 😜

I didn't even know this was one of those words that has a different spelling in US. Sorry.

**Review by caroldelwing — APPROVED** [review]
:rocket::face_holding_back_tears:

**Pull request merged**